Makefile

```diff
@@ -117,10 +117,10 @@ install-doc-deps:
 	pip install -r docs/requirements.txt
 
 install-stub-gen:
-	python -mpip install -e stub_gen
+	python -mpip install ./stub_gen
```

Comment:
Lucas can you confirm you are happy with these

Reply:
We probably should undo this change, was from debugging something. Ideally, all of this would be handled by tox anyway, need to have another look at that.